pages/apis/rest_api/analytics/runs.md

```diff
@@ -1,5 +1,29 @@
 # Runs API
 
+## List all runs
+
+Returns a [paginated list](<%= paginated_resource_docs_url %>) of runs in a test suite.
+
+```bash
+curl "https://api.buildkite.com/v2/analytics/organizations/{org.slug}/suites/{suite.slug}/runs"
+```
+
+```json
+[
+  {
+    "id": "64374307-12ab-4b13-a3f3-6a408f644ea2",
+    "branch": "main",
+    "commit_sha": "e32432c045439c95b5af2cb472b4d8be2207685d",
```

Comment:
Probably not important but I'd change this to a fake random sha. I doubt there's any security issue but just to be double sure.